src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -741,10 +765,41 @@ public void run() {
-        GT_Values.RA.addCentrifugeRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Fireclay, 19L), ItemList.Cell_Empty.get(12L), GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 4L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Silicon, 3L), GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 12L), GT_Values.NI, GT_Values.NI, GT_Values.NI, new int[]{10000, 10000, 10000}, 200, 120);
+        GT_Values.RA.addCentrifugeRecipe(CustomItemList.CokeOvenBrickDust.get(1L), GT_Utility.getIntegratedCircuit(1), GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Clay, 2L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.SiliconDioxide, 1L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Brick, 1L), GT_Values.NI, GT_Values.NI, GT_Values.NI, new int[]{10000, 9000, 7500, 6500}, 400, 30);
+        GT_Values.RA.addCentrifugeRecipe(CustomItemList.AdvancedCokeOvenBrickDust.get(1L), GT_Utility.getIntegratedCircuit(1), GT_Values.NF, GT_Values.NF, CustomItemList.CokeOvenBrickDust.get(1L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Netherrack, 2L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.SiliconDioxide, 1L), GT_OreDictUnificator.get(OrePrefixes.dustTiny, Materials.Brick, 1L), GT_Values.NI, GT_Values.NI, new int[]{10000, 9000, 7500, 6500}, 400, 120);
+        GT_Values.RA.addCentrifugeRecipe(CustomItemList.CokeOvenBrickDust.get(4L), GT_Utility.getIntegratedCircuit(2), GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Clay, 2L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Brick, 1L), GT_Values.NI, GT_Values.NI, GT_Values.NI, new int[]{10000, 9000, 7500, 6500}, 400, 120);
+        GT_Values.RA.addCentrifugeRecipe(CustomItemList.AdvancedCokeOvenBrickDust.get(36L), GT_Utility.getIntegratedCircuit(2), GT_Values.NF, GT_Values.NF, CustomItemList.CokeOvenBrickDust.get(36L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Netherrack, 36L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 9L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Brick, 4L), GT_Values.NI, GT_Values.NI, new int[]{10000, 9000, 7500, 6500}, 400, 480);
+        GT_Values.RA.addCentrifugeRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Brick, 11L), ItemList.Cell_Empty.get(4L), GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 4L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 3L), GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L), GT_Values.NI, GT_Values.NI, GT_Values.NI, new int[]{10000, 10000, 10000}, 400, 120);
```

Comment:
Isn't there a fluid output slot for centrifuge? Why put oxygen into cells?

Reply:
added a fluid oxygen variant